romancal/extern/configobj/configobj.py

```diff
@@ -0,0 +1,2476 @@
+# configobj.py
```

Comment:
We don't need to vendorize configobj in this package because it's already included in stpipe at `stpipe.extern.configobj`.  I think I may have misunderstood your question at standup -- the spec string is required here, but if we need configobj we can import it from stpipe.

Reply:
(`jwst.extern.configobj` should have been removed, an oversight on my problem which I'll open a PR to fix in a moment)

Reply:
Ah, I did not realize it's in stpipe but it totally makes sense.

Reply:
`configobj` removed